purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -44,6 +70,10 @@ data class PaywallData(
 
     @SerialName("localized_strings_by_tier") internal val localizationByTier:
     Map<String, Map<String, LocalizedConfiguration>> = emptyMap(),
+
+    @SerialName("zero_decimal_place_countries")
+    @Serializable(with = GoogleListSerializer::class)
```

Comment:
Hmm I guess this is to dereference the `google` sublevel without having to represent that in the data object right? IMO It would have been better if the backend provided the appropriate list instead of having to do this now and using more data... But we can go with this for now.